src/main/java/uk/gov/ons/census/casesvc/model/dto/FieldWorkFollowup.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.ons.census.casesvc.model.dto;
+
+import lombok.Data;
+
+@Data
+public class FieldWorkFollowup {
```

Comment:
I really don't think that it's right that Case Processor should be sending messages straight to Field. For 2021 I really want to see all the field messages get sent as actions, like every other message we send to field. This is CCS code that should really have been refactored away before doing this ticket.

Reply:
In fact, this class already exists in almost identical form, called `CcsToFwmt`